code/modules/mob/living/carbon/xenomorph/castes/ravager/abilities_ravager.dm

```diff
@@ -37,104 +37,3 @@
 	X.throw_at(A, RAV_CHARGEDISTANCE, RAV_CHARGESPEED, X)
 
 	add_cooldown()
-
-// ***************************************
-// *********** Ravage
-// ***************************************
-/datum/action/xeno_action/activable/ravage

```

Comment:
Old ravager had this ability.
This one and several others could be repurposed to work differently.